share/precommit/buildtest.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRCMAKEIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+add_test_type buildtest
+
+VERBOSE=false
+
+# files that we want to kick off
+BUILD_FILES=( build.sh )
+
+
+function buildtest_usage
+{
+  yetus_add_option "--verbose=<true|false>" "print output to console (default: false)"
+}
+
+function buildtest_parse_args
+{
+  local i
+
+  for i in "$@"; do
```

Comment:
Is it important to have the form `--verbose=true` instead of just `--verbose`?

Reply:
Good point